src/main/java/lotto/domain/model/ErrorMessages.java

```diff
@@ -0,0 +1,21 @@
+package lotto.domain.model;
+
+public enum ErrorMessages {
+    INVALID_LOTTO_NUMBER_COUNT("[ERROR] 로또 번호는 6개여야 합니다."), INVALID_NUMBER_RANGE(
+            "[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다."), INVALID_PURCHASE_AMOUNT(
+            "[ERROR] 구입 금액은 1000원 단위여야 합니다."), ZERO_PURCHASE_AMOUNT(
+            "[ERROR] 구입 금액이 0일 수 없습니다."), MINIMUM_PURCHASE_AMOUNT(
+            "[ERROR] 구입 금액은 1000원 이상이어야 합니다."), INVALID_INPUT_FORMAT("[ERROR] 잘못된 입력 형식입니다."), EMPTY_INPUT(
+            "[ERROR] 입력은 빈 값일 수 없습니다."), DUPLICATE_WINNING_NUMBER_ERROR(
+            "[ERROR] 당첨 번호는 중복될 수 없습니다."), BONUS_NUMBER_DUPLICATE_ERROR("[ERROR] 보너스 번호는 당첨 번호와 중복될 수 없습니다.");
```

Comment:
```suggestion
public enum ErrorMessages {
    INVALID_LOTTO_NUMBER_COUNT("[ERROR] 로또 번호는 6개여야 합니다."),
    INVALID_NUMBER_RANGE("[ERROR] 로또 번호는 1부터 45 사이의 숫자여야 합니다."),
    INVALID_PURCHASE_AMOUNT("[ERROR] 구입 금액은 1000원 단위여야 합니다."),
    ZERO_PURCHASE_AMOUNT("[ERROR] 구입 금액이 0일 수 없습니다."),
    MINIMUM_PURCHASE_AMOUNT("[ERROR] 구입 금액은 1000원 이상이어야 합니다."),
    INVALID_INPUT_FORMAT("[ERROR] 잘못된 입력 형식입니다."),
    EMPTY_INPUT("[ERROR] 입력은 빈 값일 수 없습니다."), 
    DUPLICATE_WINNING_NUMBER_ERROR("[ERROR] 당첨 번호는 중복될 수 없습니다."), 
    BONUS_NUMBER_DUPLICATE_ERROR("[ERROR] 보너스 번호는 당첨 번호와 중복될 수 없습니다.");
```
적절한 개행으로 보기 편하게 만들어주는건 어떨까요?
추가로 `[ERROR]`라는 prefix가 반복되고 있는데요 반복을 줄일 방법은 없을까요?

Reply:
개행 통해서 보기좋게 개선해보겠습니다. [ERROR] prefix를 공통 접두사로 처리할 방법을 고민해보겠습니다!